pkg/pwgen/pwrules/aliases_test.go

```diff
@@ -22,7 +21,7 @@ func TestLoadCustomRules(t *testing.T) {
 		require.NoError(t, cfg.Set("", "domain-alias."+k+".insteadOf", v))
 	}
 
-	ctx := context.Background()
+	ctx := config.NewContextReadOnly()
 	ctx = cfg.WithConfig(ctx)
```

Comment:
Here we're using the above config in our context anyway, so a plain bg context is fine.